### DISCUSSION TITLE
Editor can falsely warn of a difficulty with the same name while swapping names

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#21785] [Q&A] [answered]
Not sure how to always reproduce this but most of the time, when swapping names to 2 difficulties, the editor gives a false error saying that one of the two difficulties already has the new name.
I am not convinced of this, but it seems like it could also depend on how soon this swap is made

Here are the steps I made in the video:
- you can see 2 difficulties:
  - 'easy' (blue)
  - 'normal' (cyan)
- renamed 'easy' to 'hard'
- renamed 'normal' to 'easy'
- tried to rename 'hard' to 'normal'.
  the editor says that the map already has a difficulty with the name 'normal'

Note: I made one of the two difficulties using the 'exact copy' option, and then I added more objects to increase its star rating.

https://user-images.githubusercontent.com/43073074/209465564-141de3e6-01f5-4c4b-9ba6-0846386a6834.mp4

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/21794
